### ISSUE TITLE
Possible workaround A7 devices mostly

### ISSUE THREAD

**Issue by ForgottenGods** [checkra1n/BugTracker#2000]

**Comment by ForgottenGods** [comment]
I have been trying with an iPad mini 2 ios 12.5.1 , and I have found another partial workaround. I was able to replicate it as well but in my test I did it 20 times and was only able to get it to work 3 times.

What I did was when it got at "right before trigger " I rapidly unplugged and replugged my iPad it take about 3 -5 times of removing the cable then putting it back in. then I would see the "booting" line ,however most of time what happened was it just booted back to the setup menu (my device was reset to act as an icloud locked device)
But 3 out of the 20 times I tried it went through the whole process.
This is on a windows pc using bootra1n as I don't own a mac, also I'm seemed to have better success rate using the 32bit version of bootra1n ,even though my pc is 64bit

I hope this helps somebody as it seems to be a little more consistent. Have a good day

**Comment by marco-calautti** [comment]
I just used a different computer. The only difference I can see, usb-related, with my computer and the other, is that the other has USB2 ports, while mine only has USB3.
-EDIT-
I confirm that on my other laptop, checkra1n works 100% of the times without having to disconnect and reconnect the cable (it just works). This is a Ubuntu 18.04 machine, and I connected my iPad 7th gen on the USB2 port.
